### ISSUE TITLE
Feature Request - Tasks - Add ability to create and reorder statuses

### ISSUE THREAD

**Issue by mcarpenter622** [steffendx/GoNorth#97]
![image](https://user-images.githubusercontent.com/13891542/168170494-4c9fc58c-04f8-4d90-b1cc-aa68b5268129.png)

**Comment by steffendx** [comment]
Good idea. I will include something like this in one of the upcoming releases. Since I already started the current release and I am quite busy at the moment this might take a bit of time though.
